src/src/transport/uart.c

```diff
@@ -290,6 +292,7 @@ otError otPlatUartSend(const uint8_t *aBuf, uint16_t aBufLength)
 
     // Set up transmit buffer.
     sTransmitBuffer = aBuf;
+    sTransmitDone   = false;
 
```

Comment:
not aligned?

Reply:
It is aligned with the previous line. Are you checking the previous commit?